packages/core/src/tools.ts

```diff
@@ -233,6 +233,16 @@ export namespace Tools {
 		return transformMatrixArray[4];
 	}
 
+	/**
+	 * Returns an array of the translations from the element's attributes.
+	 * @param elementRef
+	 */
+	export function getTranslations(elementRef: any) {
```

Comment:
I don't think this function will work as you expect it to.

For instance, if elementRef was:
```html
<text transform="translate(50, 30) rotate(31, 80)"></text>
```

Then it'd grab the wrong thing.

Also, the function should be a little more general and less specific to `translate`. It'd be good if it separated all the transforms into an object or array, then depending on the use case we'd take the specific transform out of that.

Reply:
yeah its useful in the long run to get all the transforms